src-electron/rest/file-ops.js

```diff
@@ -41,24 +41,31 @@ const studio = require('../ide-integration/studio-rest-api')
 function httpPostFileOpen(db) {
   return async (req, res) => {
     let { zapFilePath, ideProjectPath } = req.body
-    let name = ''
+    if (req.body.search?.includes('filePath=%2F')) {
+      ideProjectPath = dbEnum.sessionKey.ideProjectPath
+      zapFilePath = req.body.search?.split('filePath=')
+      zapFilePath = zapFilePath[1]?.replaceAll('%2F', '//').trim()
+      console.log(zapFilePath)
+    }
 
+    // let { ideProjectPath } = req.body
+    let name = ''
+    //console.log(zapFilePath)
     if (zapFilePath) {
       name = path.posix.basename(zapFilePath)
       env.logInfo(`Loading project(${name})`)
 
       try {
         // set path before importDataFromFile() to avoid triggering DIRTY flag
-        if (ideProjectPath) {
-          env.logInfo(`IDE: setting project path(${name}) to ${ideProjectPath}`)
-          // store studio project path
-          await querySession.updateSessionKeyValue(
-            db,
-            req.zapSessionId,
-            dbEnum.sessionKey.ideProjectPath,
-            ideProjectPath
-          )
-        }
+        //  if (ideProjectPath) {
+        //env.logInfo(`IDE: setting project path(${name}) to ${ideProjectPath}`)
```

Comment:
ideProjectPath is here so ZAP knows how to interact with the Studio IDE.
The logInfo should stay to make debugging possible.

Reply:
agree, will fix - thanks